### PR TITLE
verbage fixes for react guide

### PR DESCRIPTION
**Pull request by VinceJuliano**
Small edits to the react starter kit

### CODE REVIEW

File: docs/src/kits/react/create-react-app.md [hunk 1]
```diff
@@ -289,8 +289,29 @@ yarn deploy
...
-If you receive this error `Not enough funds to send data`, you have to fund some AR into your wallet, and then try to deploy it again.
+::: info ERROR
+If you receive this error `Not enough funds to send data`, you have to fund some AR into your Bundlr wallet, and then try to deploy it again, run
+:::
+
+<CodeGroup>
+  <CodeGroupItem title="NPM">
+  
+```console:no-line-numbers
+bundlr fund 1479016 -h https://node1.bundlr.network -w wallet.json -c arweave
```
Comment (mrsaifullah52) [inline docs/src/kits/react/create-react-app.md:300]: just tell the user little bit about the smallest unit of the AR

**Review by mrsaifullah52 — APPROVED** [review]
Looks Perfect 🤩

File: docs/src/kits/react/create-react-app.md (hunk 1 above)
Comment (VinceJuliano) [reply docs/src/kits/react/create-react-app.md:300]: Done! thanks Saif

Comment (mrsaifullah52) [reply docs/src/kits/react/create-react-app.md:300]: Awesome @VinceJuliano

**Pull request merged**